builds/azure-pipelines/build-release-java.yml

```diff
@@ -0,0 +1,51 @@
+trigger: none
+
+pr: none
+
+variables:
+  version: '0.1.0'
```

Comment:
I'd suggest just putting this directly into the pom.xml - that's more expected for where to find the current version. (and then remove the set library version step)